core/model/modx/processors/workspace/packages/getattribute.class.php

```diff
@@ -39,8 +39,12 @@ public function initialize() {
     public function process() {
         $attributes = array();
         $attributesToGet = explode(',',$this->getProperty('attributes',''));
+        $Parsedown = new Parsedown();
```

Comment:
As we use the external tool to improve readability we also can add a little bit more security. 
`$parsedown->setSafeMode(true);` can help to avoid XSS and other bad things, because packages from repository checked but they still unsafe user input.

Reply:
I think there can't be any security issues like XSS since the text is encoded with htmlentities. Or does Parsedown replace those entities inside?